database-emulator/javascript-quickstart/test/test.js

```diff
@@ -18,10 +18,12 @@ const rules = fs.readFileSync('database.rules.json', 'utf8');
  * @return {object} the app.
  */
 function authedApp(auth) {
-  return firebase.initializeTestApp({
-    databaseName: databaseName,
-    auth: auth,
-  }).database();
+  return firebase
+    .initializeTestApp({
+      databaseName: databaseName,
+      auth: auth
+    })
```

Comment:
pro-tip that I recently learned: if the value is a variable with the same name as the key, you can just write the key. It will infer the value.

So this can be
```
return firebase.initializeTestApp({ databaseName, auth });
```